pypesto/sample/parallel_tempering.py

```diff
@@ -69,8 +70,10 @@ def initialize(
 
     def sample(self, n_samples: int, beta: float = 1.0):
         """Sample and swap in between samplers."""
+        show_progress = self.options['show_progress']
         # loop over iterations
-        for i_sample in tqdm(range(int(n_samples))):  # TODO test
+        for i_sample in tqdm(range(int(n_samples)), disable=not show_progress):
+            # TODO test
```

Comment:
Is this still ToDo?

Reply:
not sure, was there before. Would probably have to ask @yannikschaelte

Reply:
yeah I guess we wanted to add tests there, tbd independent of current pr